Makefile

```diff
@@ -257,7 +257,9 @@ update-authors:
 docs-container:
 	grep -v -E "(SOURCE|GIT)_VERSION" .gitignore >.dockerignore
 	echo ".*" >>.dockerignore # .git pruned out
-	docker image build -t cilium/docs-builder -f Documentation/Dockerfile .
+	cp -r ./api ./Documentation/_api
+	docker image build -t cilium/docs-builder -f Documentation/Dockerfile ./Documentation
+	rm -r ./Documentation/_api
```

Comment:
Let's also clean it in the `clean` target to allow cleaning up when `docker image build` has failed

Reply:
@tgraf there's no clean target for the documentation. Is my proposal acceptable?